allennlp/modules/token_embedders/pretrained_transformer_embedder.py

```diff
@@ -62,6 +62,10 @@ def __init__(
         self._num_added_end_tokens = len(tokenizer.single_sequence_end_tokens)
         self._num_added_tokens = self._num_added_start_tokens + self._num_added_end_tokens
 
+        if not self._train_parameters:
```

Comment:
Do you even need to keep this around as a member variable anymore?

Reply:
Probably not